library/src/main/java/com/schibsted/spain/barista/internal/matcher/TextMatcher.kt

```diff
@@ -0,0 +1,45 @@
+package com.schibsted.spain.barista.internal.matcher
+
+import android.view.View
+import android.widget.TextView
+import androidx.test.espresso.matcher.BoundedMatcher
+import com.google.android.material.textfield.TextInputLayout
+import org.hamcrest.Description
+import org.hamcrest.Matcher
+import org.hamcrest.core.Is
+
+fun withCompatText(string: String): Matcher<View> {
+  return withCompatText(Is.`is`<String>(string))
+}
+
+fun withCompatText(stringMatcher: Matcher<String>): Matcher<View> {
+  return withCompatTextMatcher(stringMatcher)
+}
+
+internal fun withCompatTextMatcher(stringMatcher: Matcher<String>): BoundedMatcher<View, View> {
```

Comment:
This matcher could probably be rewritten to be a composition of existing Espresso matchers instead of a copy 🤔

Reply:
done!